README.md

```diff
@@ -12,52 +12,60 @@ To interact the client certificates, application code should not need to use mos
 
 ### Quick Start
 
-1. Install gcloud CLI (Cloud SDK) at: https://cloud.google.com/sdk/docs/install.
-
-   1. **Note:** gcloud version 416.0 or newer is required.
-
-1. `$ gcloud components install enterprise-certificate-proxy`.
-
-1. **MacOS ONLY**
-
-   1. `$ gcloud config virtualenv create`
+1. Install gcloud CLI (Cloud SDK) at: https://cloud.google.com/sdk/docs/install. Install with the bundled python option enabled.
 
-   1. `$ gcloud config virtualenv enable`
+   1. **Note:** gcloud version 416.0 or newer is required. Version 430.0 or newer is recommended.
 
-1. Create a new JSON file at `~/.config/gcloud/certificate_config.json`:
-
-    - Alternatively you can put the JSON in the location of your choice and set the path to it using:
+1. For macOS and Linux, run the install.sh script after downloading it to complete installation. 
+    ```
+    $ ./google-cloud-sdk/install.sh
+    ```
+1. Install the ECP helper component:
+    ```
+    $ gcloud components install enterprise-certificate-proxy
+    ```
+1. Initialize ECP certificate configuration:
 
-      `$ gcloud config set context_aware/enterprise_certificate_config_file_path "<json file path>"`.
+   * **MacOS** `$ gcloud auth enterprise-certificate-config create macos --issuer=<CERT_ISSUER>`
 
-    - Another approach for setting the JSON file location is setting the location with the `GOOGLE_API_CERTIFICATE_CONFIG` environment variable.
+   * **Linux** `$ gcloud auth enterprise-certificate-config create linux --label=<CERT_LABEL> --module=<PKCS11_MODULE_PATH> --slot=<SLOT_ID>`
 
-1. Update the `certificate_config.json` file with details about the certificate (See [Configuration](#certificate-configutation) section for details.)
+   * **Windows** `$ gcloud auth enterprise-certificate-config create windows --issuer=<CERT_ISSUER> --provider=<PROVIDER> --store=<STORE>`
 
 1. Enable usage of client certificates through gcloud CLI config command:
     ```
-    gcloud config set context_aware/use_client_certificate true
+    $ gcloud config set context_aware/use_client_certificate true
+    ```
+1. You can now use gcloud to access CBA-protected GCP resources. For example:
+    ```
+    $ gcloud pubsub topics list
     ```
 
-1. You can now use gcloud to access GCP resources with mTLS.
+### Manual Certificate Configuration
 
-### Certificate Configuration
+ECP relies on the `certificate_config.json` file to read all the metadata information for locating the certificate.
+It is stored as a JSON file at the following location on the user's device:
+
+* **Linux and MacOS**: ~/.config/gcloud/certificate_config.json
+* **Windows**: %APPDATA%\gcloud\certificate_config.json 
+
+You can also put the JSON in the location of your choice and set the path to it using:
+
+      `$ gcloud config set context_aware/enterprise_certificate_config_file_path "<json file path>"`.
```

Comment:
Has this been implemented? A cursory check says no

Reply:
Good catch, I removed this for now.